### PR TITLE
Fix punctuation

### PR DESCRIPTION
**Pull request by Pencil-Yao**
1. fix punctuation error of markdown
2. fix pic url error

### CODE REVIEW

**Comment by rainchen** [comment]
this commit "Merge pull request #1 from citahub/master"  was not expected, please rebase your branch

File: docs/cita/architecture/architecture.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: 系统架构
 
 许可链在交易吞吐量低、交易实时性上很难满足行业应用的要求。另外，区块链技术发展迅速，单体应用也很难满足演进要求。为了解决这两个问题，CITA 将单个节点按照功解构为交易共识、合约引擎、链式存储、网络同步、服务网关等多个松耦合的微服务，一方面利用云计算基础设施来按需提升性能，另一方面，各个组件可独立替换升级。另外，采用消息总线进行通信，各个微服务可以利用异步消息简化处理，又可以确保消息的安全、可靠送达。当与外部系统集成时，外部系统可以直接访问消息总线，而无需 CITA 做适配。
 
-![](assets/cita-assets/architecture.jpg)
+![](../../assets/cita-assets/architecture.jpg)
```
Comment (rainchen) [inline docs/cita/architecture/architecture.md:10]: is this change necessary? 
The image architecture.jpg looks good in https://docs.citahub.com/zh-CN/cita/architecture/architecture .

Comment (Pencil-Yao) [reply docs/cita/architecture/architecture.md:10]: But in markdown reader, the earlier file can't display the image correctly
